samples/zigbee/light_bulb/src/zcl_scenes.c

```diff
@@ -0,0 +1,1005 @@
+/**
+ * Copyright (c) 2020, Nordic Semiconductor ASA
```

Comment:
Please use correct licence and correct year in licence

Reply:
fixed